ambry-store/src/main/java/com.github.ambry.store/BlobStoreCompactor.java

```diff
@@ -826,6 +885,9 @@ private void waitTillRefCountZero() throws InterruptedException {
           time.sleep(singleWaitMs);
           waitSoFar += singleWaitMs;
         }
+        if (srcLog.getSegment(segmentName).refCount() > 0) {
+          logger.debug("Ref count of {} in {} did not go down to 0", segmentName, dataDir);
```

Comment:
Shouldn't this be more than a DEBUG statement? Perhaps a WARN since the cleanup is going to happen regardless?